astroquery/query.py

```diff
@@ -114,7 +114,7 @@ def from_cache(self, cache_location):
                 response = pickle.load(f)
             if not isinstance(response, requests.Response):
                 response = None
-        except IOError:  # TODO: change to FileNotFoundError once drop py2 support
+        except OSError:  # TODO: change to FileNotFoundError once drop py2 support
```

Comment:
The comment here suggests using a more specific subclass of `OSError`, it might be a good idea to do the same elsewhere too.

Reply:
Hah, `sed` doesn't read the comments :)